### PR TITLE
Replace references to “(Mac) OS X” with “macOS”

### PR DESCRIPTION
**Pull request by izzynavedo**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --new-formula <formula>` (after doing `brew install <formula>`)?

---

### CODE REVIEW

File: Formula/apcupsd.rb [hunk 1]
```diff
@@ -60,7 +60,7 @@ def caveats
 
     if MacOS.version >= :el_capitan
       s += <<-EOS.undent
-        Note: On OS X El Capitan and above, the kernel extension currently
+        Note: On macOS El Capitan and above, the kernel extension currently
```
Comment (MikeMcQuaid) [inline Formula/apcupsd.rb]: When referring to a specific version we should use the correct naming e.g. OS X in this case.

File: Formula/archey.rb [hunk 2]
```diff
@@ -12,6 +12,6 @@ def install
   end
 
   test do
-    assert_match "Archey OS X 1", shell_output("#{bin}/archey --help")
+    assert_match "Archey macOS 1", shell_output("#{bin}/archey --help")
```
Comment (MikeMcQuaid) [inline Formula/archey.rb]: Are you sure?

File: Formula/ccze.rb [hunk 3]
```diff
@@ -18,7 +18,7 @@ class Ccze < Formula
   def install
     # https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=823334
     inreplace "src/ccze-compat.c", "#if HAVE_SUBOPTARg", "#if HAVE_SUBOPTARG"
-    # Allegedly from Debian & fixes compiler errors on old OS X releases.
+    # Allegedly from Debian & fixes compiler errors on old macOS releases.
```
Comment (MikeMcQuaid) [inline Formula/ccze.rb]: OS X

File: Formula/cloc.rb [hunk 4]
```diff
@@ -27,7 +27,7 @@ class Cloc < Formula
   def install
     ENV.prepend_create_path "PERL5LIB", libexec/"lib/perl5"
 
-    # These are shipped as standard with OS X/macOS' default Perl, but
+    # These are shipped as standard with macOS/macOS' default Perl, but
```
Comment (MikeMcQuaid) [inline Formula/cloc.rb]: Duplicated

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]
I've only reviewed a few places but you'll need to check them all not all changes makes sense.

**Comment by izzynavedo** [comment]
I think I've found and corrected all the errors.

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**